### PR TITLE
Removing Dead Link

### PR DESCRIPTION
**Pull request by TokyoScarab**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2418

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/what-is-phone-system-in-office-365.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To enable calls to landlines and mobile phones, Phone System can be connected to
   
 - By purchasing the a Calling Plan (domestic or domestic and international) for Office 365. See [Phone System and Calling Plans](calling-plan-landing-page.md)
     
-- By using on-premises PSTN connectivity, where software on premises connects with your existing telephony infrastructure. See [Planning your Phone System](/skypeforbusiness/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-your-phone-system-cloud-pbx-solution).
```
Comment (TokyoScarab) [inline Teams/what-is-phone-system-in-office-365.md]: See [Plan for Skype for Business Cloud Connector Edition](https://docs.microsoft.com/skypeforbusiness/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-skype-for-business-cloud-connector-edition).

Comment (nenonix) [reply Teams/what-is-phone-system-in-office-365.md]: @TokyoScarab click on the three dots at right and select **edit file** to edit your PR,

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
@scanum:  This is the correct link:  https://docs.microsoft.com/en-us/SkypeForBusiness/hybrid/msft-telephony-solutions

Please update and then I'll merge the pull request.  Thanks.

File: Teams/what-is-phone-system-in-office-365.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ To enable calls to landlines and mobile phones, Phone System can be connected to
...
+- By using on-premises PSTN connectivity, where software on premises connects with your existing telephony infrastructure. See [Plan for Skype for Business Cloud Connector Edition](https://docs.microsoft.com/skypeforbusiness/skype-for-business-hybrid-solutions/plan-your-phone-system-cloud-pbx-solution/plan-skype-for-business-cloud-connector-edition).
```
Comment (nenonix) [inline Teams/what-is-phone-system-in-office-365.md]:
```suggestion
- By using on-premises PSTN connectivity, where software on premises connects with your existing telephony infrastructure. See [Microsoft telephony solutions](https://docs.microsoft.com/SkypeForBusiness/hybrid/msft-telephony-solutions).
```

**Pull request merged**